src/cc_toolchain/BUILD

```diff
@@ -69,56 +69,57 @@ filegroup(
 )
 
 filegroup(
-    name = "clang_wrappers",
-    srcs = glob(["wrapper/clang-*"]),
+    name = "k8_gcc_wrappers",
+    srcs = glob(["wrapper/k8_gcc-*"]),
 )
 
 filegroup(
-    name = "clang_all",
+    name = "k8_gcc_all",
     srcs = [
-        ":clang_wrappers",
-        "@llvm_clang//:binaries",
-        "@llvm_clang//:clang_libs",
-        "@llvm_clang//:includes",
-        "@llvm_clang//:runtime_libs",
-        "@llvm_clang//:static_libs",
+        ":k8_gcc_wrappers",
+        "@k8_gcc//:binaries",
+        "@k8_gcc//:includes",
+        "@k8_gcc//:k8_gcc_libs",
+        "@k8_gcc//:runtime_libs",
+        "@k8_gcc//:static_libs",
```

Comment:
I think most of the stuff here is linux-specific?

Reply:
cleaned it up